app/controllers/api/v2/ansible_override_values_controller.rb

```diff
@@ -31,6 +31,23 @@ def create
         render 'api/v2/ansible_override_values/show'
       end
 
+      api :PUT, "/ansible_override_values", N_("Update an override value")
```

Comment:
I was following the `POST` method by passing the id of the variable in the body of the request with `ansible_variable_id`, rather than doing it as a part of the url.
So the only difference between the create and update requests is the method chosen (`POST or `PUT`)

Example:
Create a override:
```sh
curl -v -u "user:pass" -X POST https://foreman.example.tld/ansible/api/v2/ansible_override_values \
   -H "Content-Type: application/json" \
   -d '{
         "ansible_variable_id": "1337",
         "override_value": {
           "match": "domain=example.tld",
           "value": "value"
         }
       }'
```

Update a override:
```sh
curl -v -u "user:pass" -X PUT https://foreman.example.tld/ansible/api/v2/ansible_override_values \
   -H "Content-Type: application/json" \
   -d '{
         "ansible_variable_id": "1337",
         "override_value": {
           "match": "domain=example.tld",
           "value": "new_value"
         }
       }'
```

But I can update it so that it's like this if you prefer:

Update a override:
```sh
curl -v -u "user:pass" -X PUT https://foreman.example.tld/ansible/api/v2/ansible_override_values/1337 \
   -H "Content-Type: application/json" \
   -d '{
         "override_value": {
           "match": "domain=example.tld",
           "value": "new_value"
         }
       }'
```

Reply:
Yes, I suggest we align with RESTful principles by updating it to `PUT /ansible_override_values/:id`.